src/Kliva/Controls/ActivityFeedControl.xaml

```diff
@@ -17,6 +17,24 @@
 
     <Grid>
         <VisualStateManager.VisualStateGroups>
+            <VisualStateGroup x:Name="AdaptiveStates" CurrentStateChanged="AdaptiveStates_CurrentStateChanged">
```

Comment:
Current ActivityFeedControl.xaml has some extra setters in the VisualStateGroups ( for ActivityDetailCommandBar and ActivityMap ).
For the commandbar we play with visibility, do we need to adjust this more for the new master/detail ( like defer loading ? )